src/cmd/services/m3dbnode/config/config_test.go

```diff
@@ -457,8 +457,8 @@ func TestConfiguration(t *testing.T) {
     queueChannel: null
   repair:
     enabled: false
-    type: 0
```

Comment:
The expected yaml changed because I added the marshal functions here https://github.com/m3db/m3/pull/3903/files#diff-ad0b86af7925c9f6079fe57fff74175b298ea255d3582b298df8f9c5d33375ebR51

(I added the marshal functions because the DeepCopy function was implemented as marshaling-then-unmarshaling the db node configuration.  But unmarshalYAML is not working properly with "type: 0" and "strategy: 0".)